### PR TITLE
Fix #619 [Favourites Menu] Actionbar gets removed

### LINKED ISSUE #619: [Favourites Menu] Actionbar gets removed
Actionbar gets removed when clicked on Favourites menu.

### Steps to reproduce
* Open the App
* Click on Favourite menu.
* Click back(Phone's Button)

### Actual behaviour
There is no Actionbar on Activity after clicking back.

image

### Expected behaviour
Actionbar should be there in Activity.

### PR DESCRIPTION
**Pull request by theabhishekavi**
Fixes #619 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java [hunk 1]
```diff
@@ -92,14 +96,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         mSharedpreferences = PreferenceManager
                 .getDefaultSharedPreferences(getActivity());
         mSharedpreferences.registerOnSharedPreferenceChangeListener(this);
-        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
-        actionBar.hide();
+//        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: remove comments

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java [hunk 2]
```diff
@@ -9,7 +9,11 @@
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
 import android.support.v7.preference.PreferenceManager;
+import android.util.Log;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: remove unnecessary import

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java (hunk 1 above)
Comment (theabhishekavi) [reply app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: okay. i am still working on issue number #622 as that contains 3-4 sub parts. sorry for the delay.

File: app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java (hunk 2 above)
Comment (theabhishekavi) [reply app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java]: okay i will remove unnecessary imports and comments

**Comment by theabhishekavi** [comment]
@Swati4star Removed the unnecessary imports and comments. Kindly review now :)

**Pull request merged**